lib/acme/client.rb

```diff
@@ -315,6 +316,18 @@ def fetch_chain(response, limit = 10)
     end
   end
 
+  def fetch_alternative_links(response)
+    get_links(response, 'alternate').uniq.sort
+  end
+
+  # Attributes in the "Link" header such as `rel` are not unique (RFC8555).
+  # Therefore, it is possible that more than one alternative may be provided in the future.
+  # Currently the Link attribute is a single hash by Faraday::Middleware,
+  # but it will be returned as an Array to accommodate future changes.
```

Comment:
Do you mean that `response.headers['link']` return a hash an sometime an array?

Reply:
I've wrote about this as new issue.

https://github.com/unixcharles/acme-client/issues/188

alternate_certs should be considered for future use, as there may be more than one of them.